openapi/src/openapi-src.json

```diff
@@ -228,6 +228,101 @@
                         "$ref": "#/components/responses/NotFound"
                     }
                 }
+            },
+            "get": {
+                "tags": ["Answers"],
+                "summary": "Get answer(s) for a specified section",
+                "description": "Gets the answer(s) to a supplied section",
+                "operationId": "getAnswer",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/questionnaireId"
+                    },
+                    {
+                        "$ref": "#/components/parameters/sectionId"
+                    },
+                    {
```

Comment:
Given the all answers endpoint isn't versioned, it might be worth leaving off the versioning for the first pass